samples/Makefile

```diff
@@ -20,7 +21,7 @@ SAMPLEDIR = .
 OBJDIR = .
 LIBDIR = ..
 
-CFLAGS += -Wall -I$(INCDIR)
+CFLAGS += -Wall -Werror -Wno-unused-function -I$(INCDIR)
```

Comment:
if there is unused function, is it better to just clean it, rather than disable the warning?